### ISSUE TITLE
Support Right-To-Left languages

### ISSUE THREAD

**Issue by nuttyartist** [nuttyartist/notes#64]
In part of our translation and localization issue #10, we should devote our attention to RTL languages. This is mainly (from what I suspect) Arabic and Hebrew, if there are more common RTL languages you're welcome to comment.

Mainly 3 things needs to be done:

- [ ] Good looking Arabic and Hebrew fonts. (Should we embed free fonts or use system default?)
- [ ] Support  RTL languages in the noteView (Right now they are "sitting" from left to right).
- [x] Consider how and if to implement the changing of the position of the noteView-foldersView, without requiring a settings panel, and without cluttering the interface.

**Comment by nuttyartist** [comment]
In the future version there's gonna be a 3 dots button on the top right of the note's editor.
![image](https://cloud.githubusercontent.com/assets/16375940/15965048/41c8f5a6-2f24-11e6-8573-bd552ad078a8.png)
We may want to put an option for changing the interface to RTL in that. way:
View -> Right to left , Always on top (Not quite sure about these).

What do you think?

**Comment by theshadowx** [comment]
I think ... should be top (same line as "x-+") as it's linked to the GUI and not just the text. And leave all  icons that has a link with just the textfield apart.

**Comment by nuttyartist** [comment]
Adding this option as a button that way will clutter the interface. We really need to strive for simplicity - and that is making only the necessary visible on the interface. Changing the position of the noteView-foldersView from left-to-right or right-to-left is something the user won't do frequently, maybe even done only once in the entire time they will use the app. So we would stick with an option in the 3-dots menu to for doing that.

**Comment by nuttyartist** [comment]
Closing this, and putting everything [here](https://github.com/nuttyartist/notes/issues/10).
